configs/cardano/private/download.sh

```diff
@@ -0,0 +1,10 @@
+#! /bin/bash
+
+set -euo pipefail
+
+wget https://book.play.dev.cardano.org/environments/private/config.json
+wget https://book.play.dev.cardano.org/environments/private/topology.json
+wget https://book.play.dev.cardano.org/environments/private/byron-genesis.json
+wget https://book.play.dev.cardano.org/environments/private/shelley-genesis.json
+wget https://book.play.dev.cardano.org/environments/private/alonzo-genesis.json
+wget https://book.play.dev.cardano.org/environments/private/conway-genesis.json
```

Comment:
```suggestion
wget https://book.play.dev.cardano.org/environments/private/conway-genesis.json

```

(newline at end of file)